components/brave_wallet/browser/fil_tx_manager.cc

```diff
@@ -309,6 +309,7 @@ void FilTxManager::OnGetNextNonceForHardware(
   auto message = meta->tx()->GetMessageToSign();
   if (!message.has_value()) {
     std::move(callback).Run(nullptr);
+    return;
```

Comment:
Similar to import above, this is changing the behavior. It looks like there was a bug if the callback was called twice. But wanted to raise in case we've worked around that bug or expect the broken behavior
cc: @darkdh

Reply:
Like I've explained above `base::OnceCallback` cannot be run twice, so the second call to run will cause a crash.

Reply:
It's a bug here, we should have the return.